airflow/www/static/js/dag/useSelection.test.tsx

```diff
@@ -38,17 +38,19 @@ describe('Test useSelection hook', () => {
       selected: {
         runId,
         taskId,
+        mapIndex,
       },
     } = result.current;
 
     expect(runId).toBeNull();
     expect(taskId).toBeNull();
+    expect(mapIndex).toBeNull();
   });
 
   test.each([
-    { taskId: 'task_1', runId: 'run_1' },
-    { runId: 'run_1', taskId: null },
-    { taskId: 'task_1', runId: null },
+    { taskId: 'task_1', runId: 'run_1', mapIndex: 2 },
+    { runId: 'run_1', taskId: null, mapIndex: null },
+    { taskId: 'task_1', runId: null, mapIndex: 1 },
```

Comment:
Let's add a test case for `mapIndex: 0` and `mapIndex: -1`.

For -1, we probably need to cast it to null. Currently, I don't see anywhere this pops up in the UI, but a test couldn't hurt.

Reply:
Done :)